.gitignore

```diff
@@ -6,7 +6,6 @@
 .DS_Store
```

Comment:
(General comment)

Do we want to take the time this sprint to incorporate GitHub's recommended .gitignore for python projects?
https://github.com/github/gitignore/blob/master/Python.gitignore

Reply:
Sure, but let's make that a separate tech debt item that someone can move through as a quick win. Maybe we add additional labels for tech debt that we think can be accomplished in one or a few hours.